crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -1493,11 +1532,12 @@ pub unsafe fn aqc_assign_net_identifier(
 ///
 /// @relates AranyaClient.
 pub unsafe fn aqc_remove_net_identifier(
-    client: &mut Client,
+    client: *mut Client,
     team: &TeamId,
     device: &DeviceId,
     net_identifier: NetIdentifier,
 ) -> Result<(), imp::Error> {
+    let mut client = aranya_capi_core::try_as_ref!(client).try_lock()?;
     let client = client.imp();
```

Comment:
Two `let client` in a row is a bit awkward.
```suggestion
    let mut client = aranya_capi_core::try_as_ref!(client).try_lock()?.imp();
```

Reply:
It needs to be separate or else the guard gets dropped since it is just a temporary.